packages/fleet/lib/supervisor/supervisor.integration.test.ts

```diff
@@ -59,7 +59,7 @@ describe('Supervisor', () => {
             void supervisor2.start();
 
             await vi.waitUntil(() => tickSpy1.mock.calls.length > 0, {
-                timeout: 5000
+                timeout: 10000
```

Comment:
Had this one fail on me in dev again, so I went ahead and bumped it up a good bit more.